src/pages/api/companies/[id]/index.ts

```diff
@@ -118,4 +124,43 @@ const handlePUT = async (
   }
 }
 
+const handleDELETE = async (
+  req: NextApiRequest,
+  res: NextApiResponse<CompanyArchiveApiResponse>
+) => {
+  try {
+    const archivedCompany = await archiveCompany(req.query.id as string)
```

Comment:
Please re-format code like this
<img width="744" alt="Captura de Tela 2023-05-06 às 1 58 17 PM" src="https://user-images.githubusercontent.com/335095/236617405-48fecf81-2cd3-4031-98a4-a6eeb44198bc.png">